src/CMakeLists.txt

```diff
@@ -990,23 +925,9 @@ if(ALICEVISION_BUILD_SFM)
     message(STATUS "FLANN: " ${FLANN_VERSION} " (external)")
   endif()
 
-  if(ALICEVISION_USE_INTERNAL_CLP)
-    message(STATUS "CLP: " ${CLP_VERSION} " (internal)")
-  else()
-    message(STATUS "CLP: " ${CLP_VERSION} " (external)")
-  endif()
-
-  if(ALICEVISION_USE_INTERNAL_COINUTILS)
-    message(STATUS "COINUTILS: " ${COINUTILS_VERSION} " (internal)")
-  else()
-    message(STATUS "COINUTILS: " ${COINUTILS_VERSION} " (external)")
-  endif()
-
-  if(ALICEVISION_USE_INTERNAL_OSI)
-    message(STATUS "OSI: " ${OSI_VERSION} " (internal)")
-  else()
-    message(STATUS "OSI: " ${OSI_VERSION} " (external)")
-  endif()
+  message(STATUS "CLP: " ${CLP_VERSION} "")
+  message(STATUS "COINUTILS: " ${COINUTILS_VERSION} "")
+  message(STATUS "OSI: " ${OSI_VERSION} "")
```

Comment:
probably?
```suggestion
  message(STATUS "CLP: " ${Clp_VERSION} "")
  message(STATUS "COINUTILS: " ${CoinUtils_VERSION} "")
  message(STATUS "OSI: " ${osi_VERSION} "")
```

Reply:
https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-version-selection

Reply:
I confirm that it must be the same case as the name in the `find_package`, so `${Osi_VERSION}, ${Clp_VERSION}, ${CoinUtils_VERSION}`

Reply:
It works fine, thanks.